apps/website/src/components/Playground/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+import { useColorMode } from '@docusaurus/theme-common';
+import AppCode from '!!raw-loader!./template/App.js';
+import StylesCode from '!!raw-loader!./template/Styles.js';
+
+export default function Playground() {
+  const { isDarkTheme } = useColorMode();
+  const sandpackTheme = isDarkTheme ? 'sandpack-dark' : 'github-light';
+  return (
+    <Sandpack
+      template="react"
+      theme={sandpackTheme}
+      files={{ '/App.js': { code: AppCode, hidden: true }, '/Styles.js': { code: StylesCode, active: true } }}
+      customSetup={{ dependencies: { '@griffel/core': 'latest', 'highlight.js': 'latest', prettier: 'latest' } }}
+      options={{ editorHeight: 450 }}
+    />
```

Comment:
Let's switch to more complex config (we will need to do this sooner or later):

```tsx
    <SandpackProvider
      template="react"
      customSetup={{
        dependencies: { '@griffel/core': 'latest', 'highlight.js': 'latest', 'js-beautify': 'latest' },
        files: { '/App.js': { code: AppCode, hidden: true }, '/Styles.js': { code: StylesCode, active: true } },
      }}
    >
      <SandpackLayout theme={sandpackTheme}>
        <SandpackCodeEditor />
        <SandpackPreview showOpenInCodeSandbox={false} />
      </SandpackLayout>
    </SandpackProvider>
```

P.S. This also removes "export to CodeSandbox" button as it's non sense in this case.